CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 - Alpine, CentOS and Ubuntu based Docker resources for WSO2 Enterprise Integrator version `7.1.x`
+- Introduce Alpine, CentOS and Ubuntu based Docker resources for WSO2 Analytics Dashboard `1.0.0`
```

Comment:
@ThilinaManamgoda let's add the list of profiles for which you created Docker resources in the entry `Alpine, CentOS and Ubuntu based Docker resources for WSO2 Enterprise Integrator version 7.1.x` without having a new entry for Dashboard only.